main/res/drawable/zoomin.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item android:drawable="@drawable/white_bcg"/>
+    <item
```

Comment:
It's always a good idea to give visual touch feedback. Maybe adding a new layer `<item android:drawable="?android:attr/selectableItemBackground"/>` between background and icon?

GMap is doing that as well:
![grafik](https://user-images.githubusercontent.com/64581222/135758656-d8606d1e-53a9-4d73-b182-06690366bd95.png)

Reply:
Thanks for reviewing this PR.

I've tried adding the selectableItemBackground to the `zoomin.xml` in all three possible positions, but it does not seem to make any difference - no visual effects on hitting the "+" sign.

Reply:
Hm... At other places in c:geo `selectableItemBackground` works fine... 🤷‍♂️

Reply:
Maybe it does only work for layout files, not for drawables.

There are other possibilities, see https://stackoverflow.com/questions/43284336/shaped-drawable-with-selectableitembackground-as-background

But maybe it's not really worth the effort...

Reply:
Finally... Got to make it work, thanks for the link! `ripple` needed to be applied to the background XML, and the ripple color to be adjusted to `@android:color/black` instead of `?colorControlHighlight`.